components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/SCIMCommonConstants.java

```diff
@@ -192,6 +194,18 @@ public static Map<String, String> getGroupAttributeSchemaMap() {
         return groupAttributeSchemaMap;
     }
 
+    public enum dateFormats {
```

Comment:
Shall we rename this to DateFormats, since we usually start enum definitions with a capital letter?

Reply:
https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/728/commits/8663e0937f26ca103404ec414c8482a50539ca0f